Userland/Libraries/LibWeb/Geometry/DOMPoint.cpp

```diff
@@ -20,6 +21,11 @@ DOMPoint::DOMPoint(JS::Realm& realm, double x, double y, double z, double w)
     set_prototype(&Bindings::cached_web_prototype(realm, "DOMPoint"));
 }
 
+JS::NonnullGCPtr<DOMPoint> DOMPoint::from_point(JS::VM& vm, Web::Geometry::DOMPointInit dom_point_init)
+{
+    return construct_impl(*vm.current_realm(), dom_point_init.x, dom_point_init.y, dom_point_init.z, dom_point_init.w);
```

Comment:
`// The fromPoint(other) static method on DOMPoint must create a DOMPoint from the dictionary other.`